overlays/nodejs/s3/s3Mixins.ts

```diff
@@ -186,17 +185,27 @@ process.on("beforeExit", () => {
 // Mixin event handling functionality onto Bucket.
 declare module "./bucket" {
     interface Bucket {
+        /**
+         * Creates a new subscription to events fired from this Bucket to the handler provided,
+         * along with options to control the behavior of the subscription.  The handler will be
+         * called whenever a matching [s3.Object] is created.
+         */
         onObjectCreated(
             name: string, handler: BucketEventHandler,
             args?: ObjectCreatedSubscriptionArgs, opts?: pulumi.ResourceOptions): BucketEventSubscription;
 
+        /**
+         * Creates a new subscription to events fired from this Bucket to the handler provided,
+         * along with options to control the behavior of the subscription.  The handler will be
+         * called whenever an matching [s3.Object] is removed.
+         */
         onObjectRemoved(
             name: string, handler: BucketEventHandler,
             args?: ObjectRemovedSubscriptionArgs, opts?: pulumi.ResourceOptions): BucketEventSubscription;
 
         /**
-         * Creates a new subscription to the given bucket using the lambda provided, along with
-         * optional options to control the behavior of the subscription.  This function should be
+         * Creates a new subscription to events fired from this Bucket to the handler provided,
+         * along with options to control the behavior of the subscription.  This function should be
          * used when full control over the subscription is wanted, and other helpers (like
          * onObjectCreated/onObjectRemoved) are not sufficient.
          */
```

Comment:
bikeshedding. how does .onEvent sound to people here?   we could say .onBucketChanged, but i don't think there's high value in changing hte name.